views/components/HeroCard/HeroCard.css

```diff
@@ -0,0 +1,20 @@
+.HeroCard_container {
+  box-shadow: 0 0 0 0;
+  width: 565px;
+  height: 270px;
+}
+.HeroCard_image {
+  height: 106px;
+  width: 476px;
```

Comment:
can we remove these heights and widths? Maybe we need to regroup on this. If these are meant to be components that can in theory be used elsewhere in app then this could really cause us headaches.

Reply:
It's not used anywhere else besides the layout.